packages/admin/src/common/toolbar/breadcrumb/ToolbarBreadcrumbs.tsx

```diff
@@ -12,19 +12,34 @@ const BreadcrumbLink = React.forwardRef<HTMLAnchorElement, RouterLinkProps>(({ h
 
 export const ToolbarBreadcrumbs: React.FunctionComponent = () => {
     const themeProps = useThemeProps();
+    const classes = useStyles();
 
     return (
         <StackApiContext.Consumer>
             {(stackApi) => {
                 return (
                     <>
-                        {stackApi?.breadCrumbs.map(({ id, url, title }) => {
+                        {stackApi?.breadCrumbs.map(({ id, url, title }, index) => {
+                            const showSeparator = index < stackApi?.breadCrumbs.length - 1;
+                            const isActive = index === stackApi?.breadCrumbs.length - 1;
                             return (
-                                <ToolbarItem key={id}>
-                                    <Link to={url} component={BreadcrumbLink} color={"inherit"}>
-                                        <Typography {...themeProps.typographyProps}>{title}</Typography>
-                                    </Link>
-                                </ToolbarItem>
+                                <>
+                                    <div className={classes.item} key={id}>
+                                        <Link to={url} component={BreadcrumbLink} color={"inherit"}>
+                                            <Typography
+                                                {...themeProps.typographyProps}
+                                                classes={{ root: isActive ? classes.typographyActiveRoot : classes.typographyRoot }}
+                                            >
+                                                {title}
+                                            </Typography>
+                                        </Link>
```

Comment:
The `Link` should probably be inside the `Typography`, otherwise the color of the link-underline (e.g. on hover) is not the same color as the `Typography`. 

```suggestion
                                        <Typography
                                            {...themeProps.typographyProps}
                                            classes={{ root: isActive ? classes.typographyActiveRoot : classes.typographyRoot }}
                                        >
                                            <Link to={url} component={BreadcrumbLink} color={"inherit"}>
                                                {title}
                                            </Link>
                                        </Typography>
```

Reply:
thats right, looks better 👍 

![Screen Recording 2021-05-21 at 10 02 23 mov](https://user-images.githubusercontent.com/6098356/119103702-c06a3000-ba1b-11eb-872b-12ce3b0771e4.gif)